webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaVersionDiff.jsx

```diff
@@ -0,0 +1,159 @@
+/**
+  * Copyright 2017 Hortonworks.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *   http://www.apache.org/licenses/LICENSE-2.0
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+**/
+
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import Select from 'react-select';
+import Utils from '../../../utils/Utils';
+import ReactCodemirror from 'react-codemirror';
+import CodeMirror from 'codemirror';
+import 'codemirror/mode/javascript/javascript';
+import jsonlint from 'jsonlint';
+import lint from 'codemirror/addon/lint/lint';
+import {diffview} from '../../../libs/jsdifflib/diffview';
+import {difflib} from '../../../libs/jsdifflib/difflib';
+import '../../../libs/jsdifflib/diffview.css';
```

Comment:
We are importing all the css files into main.js file, can you please add this over there.